core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -135,6 +154,18 @@ public void configure() throws ConfigurationException {
 	public void open() throws FileSystemException {
 		super.open();
 		basefolderId = getBaseFolderId(getMailAddress(),getBaseFolder());
+
+		if( StringUtils.isNotEmpty(getTenantId()) ){
+			try {
+				client = ConfidentialClientApplication.builder(
+						getClientId(),
```

Comment:
Dit gaat nu nog niet helemaal goed, het moet immers ook mogelijk zijn om het clientId uit een auth alias te halen.

Reply:
Je kan ook clientId helemaal gelijk laten zijn aan username, en clientSecret aan password.
De keuze tussen moderne en ouderwetse authenticatie kan je dan maken op basis van tenantId.

Reply:
Ik snap het, client ID is een soort verkapte username en secret een verkapt password.

De bijbehorende IbisDoc van clientId en secret dan toevoegen aan username en password?

Reply:
inderdaad